includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -1591,6 +1591,30 @@ public static function add_admin_notices() {
 		}
 	}
 
+	/**
+	 * Returns JSON-formatted error details for an error term.
+	 *
+	 * @param WP_Term $term The term.
+	 * @return string Encoded JSON.
+	 */
+	public static function get_error_details_json( $term ) {
+		$json = json_decode( $term->description, true );
+
+		// Convert the numeric constant value of the node_type to its constant name.
+		$xml_reader_reflection_class = new ReflectionClass( 'XMLReader' );
+		$constants                   = $xml_reader_reflection_class->getConstants();
+		foreach ( $constants as $key => $value ) {
+			if ( $json['node_type'] === $value ) {
+				$json['node_type'] = $key;
+				break;
+			}
+		}
+
+		$json['status'] = (bool) ( (int) $term->term_group & self::ACCEPTED_VALIDATION_ERROR_BIT_MASK ) ? __( 'Removed', 'amp' ) : __( 'Kept', 'amp' );
+
+		return wp_json_encode( $json, JSON_PRETTY_PRINT );
```

Comment:
Since the slashes aren't needed in this context (and they impair readability):

```suggestion
		return wp_json_encode( $json, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES );
```

Reply:
I suppose this is not needed if the JSON is re-parsed and re-serialized. If that's the case, then `JSON_PRETTY_PRINT` isn't needed either.

Reply:
Updated in c83dbb81901fdfb3d1edd6ba0fc626354e979ed7 and 54ecc46aa9b77af20658677cb5fbc6497d360b2f